app/serializers/question_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class QuestionSerializer < ActiveModel::Serializer
+  attributes :id
+  attribute :title, key: :name
```

Comment:
seems weird to rename this to `name` in the serialization.  `title` seems more accurate to what it represents and increases clarity between the front and back ends